doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -818,6 +818,7 @@ Libraries for networking
     * The handling of MQTT JITP device association to improve speed and reliability.
     * To use nRF Cloud's custom MQTT topics instead of the default AWS topics.
     * MQTT and CoAP transports to use a single unified DNS lookup mechanism that supports IPv4 and IPv6, fallback to IPv4, and handling of multiple addresses returned by :c:func:`getaddrinfo`.
+    * To use ``CONFIG_NRF_CLOUD_FOTA_LOG_LEVEL`` instead of ``CONFIG_NRF_CLOUD_LOG_LEVEL`` in the :file:`nrf_cloud_fota_common.c` file.
```

Comment:
Since these start with `CONFIG`, aren't these Kconfig options as well?

Reply:
updated... it is just indicating a change in log module name.